packages/runtime/src/prop-controllers/instances.ts

```diff
@@ -86,19 +86,17 @@ export abstract class PropController<T = PropControllerMessage> {
     this.send = send
   }
 
-  abstract recv(message: T): void
-}
-
-class DefaultPropController extends PropController {
```

Comment:
This change makes us lose type safety. Why change the generic `T`, which is specific for each control to a generic `PropControllerMessage`?

Reply:
Hmm. I was thinking that we'd be able to define the base interface type as the more general type using generics and then we'd refine it in each control that implements the base control. I don't have a specific way in mind of how this would work since I don't have the latest state of the code in my head but I'm happy to look at it if you want.

Reply:
Okay, for now, I dropped the commit and I type cast the `recv` instead. I put a `TODO` there so we can fix it later.